employee-onboarding-A2A-reference-implementation-method-v2/agents/booking_agent_adk/__main__.py

```
@@ -0,0 +1,229 @@
+            # Step 1: Get actor token via 3-step flow using THIS agent's own client_id/secret.
+            # Step 2: Exchange the pre-scoped token using own credentials + actor token.
+            try:
+                from src.auth.asgardeo import get_asgardeo_client
+                from src.log_broadcaster import log_and_broadcast
+                asgardeo = get_asgardeo_client()
+                actor = await asgardeo._fetch_agent_actor_token(
+                    client_id=self._client_id,
+                    client_secret=self._client_secret,
+                    agent_id=self._agent_id,
+                )
+                log_and_broadcast(f"\n[BOOKING_AGENT_ACTOR_TOKEN]:")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)